### PR TITLE
Fallback to English translations in e2e tests

### PR DESCRIPTION
**Pull request by tstirrat**
Fixes e2e when there are test on new strings. It will fall back to English for any missing keys, which matches the i18n lib's functionality

#### How to test

e2e should pass

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,6 +57,7 @@
     "emotion-theming": "^10.0.27",
     "i18next": "^19.3.3",
     "js-yaml": "^3.13.1",
+    "lodash": "^4.17.15",
```
Comment (efalkner) [inline package.json:60]: wow, I can't believe we didn't have this yet

File: e2e/helpers/language.js [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
 import * as haitian from '../../app/locales/ht.json';
 
+/**
+ * Language strings for each locale. Strings are merged with english as a base
+ * to emulate i18next's english fallback in the case of missing locale strings.
+ */
 const languageStrings = {
   'en-US': english,
-  'ht-HT': haitian,
+  'ht-HT': merge({}, english, haitian),
```
Comment (Patrick-Erichsen) [inline e2e/helpers/language.js:12]: How would you feel about moving this `merge` logic down into the map function? e.g.

```js
export const languages = Object.keys(languageStrings).map(locale => [
  locale,
  merge({}, english, languageStrings[locale]),
]);
```

Comment (tstirrat) [reply e2e/helpers/language.js:12]: Way better, will do

File: package.json (hunk 1 above)
Comment (tstirrat) [reply package.json:60]: Yeah, it was only a transitive dep before.

**Pull request merged**